tests/functional/src/main/java/org/mule/functional/classloading/isolation/builder/IsolatedClassLoaderExtensionsManagerConfigurationBuilder.java

```diff
@@ -38,17 +38,17 @@
     private static Logger LOGGER = LoggerFactory.getLogger(IsolatedClassLoaderExtensionsManagerConfigurationBuilder.class);
 
     private final ExtensionManagerAdapterFactory extensionManagerAdapterFactory;
-    private final List<ArtifactClassLoader> pluginsClassLoaders;
+    private final List<ClassLoader> pluginsClassLoaders;
```

Comment:
No, I have just confirmed with @damiansima that this is not necessary. I'll revert it and use ArtifactClassLoader.